server.js

```diff
@@ -0,0 +1,31 @@
+var express = require('express');
+var { graphqlHTTP } = require('express-graphql');
+var { graphql, buildSchema } = require('graphql');
```

Comment:
`var` não é muita mais utilizado, para variáveis que não vão ser retribuídas é preferível usar `const` (isso serve pra todo o código)
```suggestion
const express = require('express');
const { graphqlHTTP } = require('express-graphql');
const { graphql, buildSchema } = require('graphql');
```